docker-compose/23.N-docker-compose.yml

```diff
@@ -69,7 +77,7 @@ services:
         http://shared-file-store:8099/alfresco/api/-default-/private/sfs/versions/1/file
     ports:
       - "8095:8095"
-    links:
```

Comment:
💘

Reply:
but for consistency I would raise another PR to do the same for all the composes we have (epic OPSEXP-2256)

Reply:
will create a new ticket in this epic